src/main/java/org/mockito/session/MockitoSessionBuilder.java

```diff
@@ -30,17 +30,47 @@
      * Migrate from {@link MockitoAnnotations#initMocks(Object)}
      * to {@link MockitoSession}!
      * <p>
+     * This method may be called multiple times to add multiple, e.g. nested, test class instances.
+     * <p>
      * See code sample in {@link MockitoSession}.
      *
      * @param testClassInstance test class instance that contains fields with Mockito annotations to be initialized.
-     *  Passing {@code null} is permitted and will make the session use a default value.
-     *  The current default is '{@code new Object()}'.
+     *  Passing {@code null} is permitted but will be ignored.
      * @return the same builder instance for fluent configuration of {@code MockitoSession}.
      * @since 2.7.0
      */
     @Incubating
     MockitoSessionBuilder initMocks(Object testClassInstance);
 
+    /**
+     * Adds the test class instances for initialization of fields annotated with Mockito annotations
+     * like {@link org.mockito.Mock}.
+     *
+     * @param testClassInstances test class instances that contains fields with Mockito annotations to be initialized.
+     *  Passing {@code null} or an empty array is permitted but will be ignored.
+     * @return the same builder instance for fluent configuration of {@code MockitoSession}.
+     * @see #initMocks(Object)
+     * @since 2.13.4
+     */
+    @Incubating
+    MockitoSessionBuilder initMocks(Object... testClassInstances);
+
+    /**
+     * Configures the name of the {@code MockitoSession} instance.
```

Comment:
It would be great to explain when the user is interested using this method. We can say that it is for framework integrations, example JUnit5, and that we don't expect the users calling this method.

Reply:
Sure!